lib/emitter.js

```diff
@@ -9,7 +9,9 @@ exports.fragment = function (doc) {
     return '';
   }
 
-  return doc[0].contents.map(emit).join('');
+  return doc.reduce(function (arr, para) {
```

Comment:
Pff, you're just a functional hater. :stuck_out_tongue:

I'm inclined to leave it as it's unlikely to ever become a bottleneck. But feel free to clean it up if you like :)